goss-testing/scripts/postgres_replication_lag.sh

```diff
@@ -25,19 +62,54 @@ do
     first_member="$(kubectl get pod -n $c_ns -l "cluster-name=$c_name,application=spilo" \
                   -o custom-columns=NAME:.metadata.name --no-headers | head -1)"
 
-    for lag in $(kubectl -n $c_ns exec $first_member -- patronictl list --format json 2>/dev/null | jq '.[]."Lag in MB"')
-    do
-        if [[ $lag != "\"\"" ]] && [[ $lag == "\"unknown"\" || $lag -gt 50 ]]; 
-        then 
-            if [[ $print_results -eq 1 ]]; 
-            then 
-                echo "Warning: $c_name has Lag: $lag"; 
-                failFlag=1 
+    echo -n "$c_name - "
```

Comment:
Shouldn't this echo only happen if -p was used?

Reply:
Yes, changed to only print with -p